packages/manager/apps/pci-gateway/.eslintrc.cjs

```diff
@@ -15,6 +15,8 @@ module.exports = {
   rules: {
     'import/extensions': 'off',
     '@typescript-eslint/no-non-null-assertion': 'off',
-    'react/no-deprecated': 'off'
+    'react/no-deprecated': 'off',
```

Comment:
why put this rule at off ?

Reply:
old rules for reactFormatter used with OsdsDatagrid